governance.md

```diff
@@ -0,0 +1,101 @@
+
+### Scope
+
+The scope of the Project includes software development under an OSI-approved open source licence supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source software project.
+
+NRE Labs has two main components: the Platform (“Antidote”), and the Lessons. The Platform can serve a wide variety of purposes as a lesson delivery platform, and similarly, Lessons may be written for a wide variety of subjects which do not directly pertain to the Mission of the project. 
+
+NRE Labs fully welcomes and encourages experimentation with both components, regardless of direct relevance to the documented Mission. However, in order to ensure that sufficient resources are consistently dedicated to fulfilling the Mission of the project, and to maintain brand consistency for Users, the Technical Community will document and maintain a core roadmap for the Platform as well as criteria for Lessons that will be hosted on labs.networkreliability.engineering. Committers will determine if Platform contributions are “Core” or “Experimental” in relation to the Community-approved roadmap and prioritize commits accordingly. Committers will also determine if Lessons meet hosting criteria. See “How to Contribute” guides for more detail. 
+
+## Technical Community
+
+The Technical Community will be responsible for all aspects of oversight relating to coordinating the technical direction of the Project, including the architecture and projects needed to achieve the Mission and Scope of the Project; for approving project or system proposals (including, but not limited to, incubation, deprecation, and changes to a sub-project’s scope) in accordance with the Contribution Process and Inbound Code Review Policy to be developed, approved and maintained by the TC; coordinating Technical Community engagement with the end-user community with respect to requirements, high level architecture, implementation experiences, use cases, etc; working with other open source or open standards communities as needed; and establishing and enforcing community norms, workflows, issuing releases, and security issue reporting policies.
+
+Technical participation in the Project is open to anyone so long as they abide by the terms of this document and documents referenced herein. Technical Community meetings are conducted in English and are open to the public and recorded. The Technical Community generally will involve Contributors and Committers. Contributors include anyone in the Technical Community who contributes code, documentation, lessons or other artifacts to the Project. Committers are Contributors who have earned the ability to modify (“commit”) source code, documentation or other artifacts in a project’s repository. Each Project will ideally have at least 2 Committers. 
```

Comment:
Not sure recording of the TC meetings should be mandated upfront in this doc. Documented at least, maybe ? Yay for transparency, but should all verbatims be systematically made available ?